documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,45 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+* With this step you can store one or more key value pairs of data stored in a group called a map or key value map.
+* To consume the ApiKeyValueMapUpload step, proceed as follows:
```

Comment:
What about @LauraVeinberga 's [comment](https://github.com/SAP/jenkins-library/pull/3407/#discussion_r792590818) on not having bullet points here.

Reply:
fixed it